apps/modernization-ui/src/validation/date/isBefore.ts

```diff
@@ -0,0 +1,17 @@
+import { internalizeDate } from 'date';
+import { isAfter } from 'date-fns';
+
+const isBefore =
+    (max: string) =>
+    (current: string): string | undefined => {
+        if (max && current) {
+            const maxDate = new Date(max);
+            const currentDate = new Date(current);
+
+            if (isAfter(currentDate, maxDate)) {
+                return `Date cannot be after: ${internalizeDate(maxDate)}`;
```

Comment:
super nit:  isn't `internalizeDate(maxDate)` the same result as the `max` parameter?